.github/workflows/androidTestSuite.yml

```diff
@@ -0,0 +1,133 @@
+name: Android Test Suite
+
+# The workflow will start for every PR against master branch
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+# Multiple jobs run in parallel
+jobs:
+  android_test_cases:
+    name: Android Unit Tests
+    runs-on: ubuntu-latest
```

Comment:
better to mention a specific OS version, e.g `ubuntu-20.04` cuz it may break on the latest releases.